packages/terra-form-field/src/Field.scss

```diff
@@ -13,10 +13,13 @@
   }
 
   .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Helvetica Neue');
```

Comment:
This does not need to be added. This style will be set by the Base component that wraps Field

Reply:
[cerner-consumer-theme](https://github.cerner.com/terra/cerner-consumer-theme/pull/1/files#diff-967ea18b58462e0c403edefef7851abfR17) requires font-family 'Roboto' 

do you mean consumer should set the  required font family in base component and we should not worry about font-family at form-field level ?

Reply:
Yes the font will be inherited from the Base component. I thought the same thing when I created these variables :)